src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -328,7 +332,6 @@ export default {
 </script>
 
 <style lang="postcss" scoped>
```

Comment:
is postcss still required? affecting some performance if added?

Reply:
Postcss should only affect the compilation, so I don't think we have to worry about removing.